doc/examples.xml

```diff
@@ -309,3 +309,35 @@ gap> GeneralisedPetersenGraph(IsMutableDigraph, 9, 4);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="LollipopGraph">
+<ManSection>
+  <Oper Name="LollipopGraph" Arg="[filt, ]m, n"/>
+  <Returns>A digraph</Returns>
+  <Description>
+    If <A>m</A> and <A>n</A> are positive integers, 
+    then this operation returns the <A>(m, n)</A>-lollipop graph
+    that consists of a complete digraph on <A>m</A> vertices and a 
+    path digraph on  <A>n</A> vertices connected by a bridge.
+
+    From
+    <URL>https://en.wikipedia.org/wiki/Lollipop_graph</URL>:
```

Comment:
Why there is a colon after the URL - is there a text after which is missing?

Reply:
Thanks @alex-konovalov probably a typo, fixed this!